### DISCUSSION TITLE
how can i locally host a mempool with mainnet and testnet but without blockstream/electrs (its heavy af)

### DISCUSSION THREAD

**Discussion by antonilol** [mempool/mempool#1138] [Q&A] [answered]
(title)

with a dropdown like this
![image](https://user-images.githubusercontent.com/49868160/149477729-a1e38c13-5f82-49f1-9ea6-1f0d4d275605.png)

**Comment by wiz** [comment]
did you read the top-level README? recommended backend is romanz/electrs for home users

**Reply by antonilol** [reply]
Yes i read it.
I already have mainnet and testnet electrs running in my node just wondered how i can use both networks on my local mempool

**Reply by antonilol** [reply]
(and txindex on both)

**Comment by wiz** [comment, marked as answer]
the title of the question was about the backend, but I think your actual question was about the `mempool-frontend-config.json` file which sets the build-time options for the frontend - if you enable multiple networks in that file and rebuild the frontend you will have a pull-down box to select the network

**Reply by antonilol** [reply]
My question is about both. Do i understand it correct that i have to run 2 backend instances (main and test) and one frontend?

**Reply by antonilol** [reply]
How can i run a mainnet mempool backend and a testnet one with one backend config and with romanz/electrs

**Reply by wiz** [reply]
Yes, this is the production instance configuration, so you will need 2 completely separate backend instances which are configured for their networks and listening on different ports - then take a look at the production nginx.conf examples for how the backend ports are proxied etc.

**Reply by Asterix89** [reply]
> the title of the question was about the backend, but I think your actual question was about the `mempool-frontend-config.json` file which sets the build-time options for the frontend - if you enable multiple networks in that file and rebuild the frontend you will have a pull-down box to select the network

**Reply by Asterix89** [reply]
Hallo I just created an account .Can someone tell me how to get intouch with mempool organisation itself?
